### DISCUSSION TITLE
How can i create DropDown type?

### DISCUSSION THREAD

**Discussion by GuiPerim** [react-page/react-page#1191] [Q&A] [answered]
So i'm pretty new using react-page, and i'm creating a custom plugin that the user can set some information.
I'd like to know if there is a way to create a field as a dropdown (see image bellow). And if so, can someone put me in the right direction?

For sample the "Company size" field, i'd like to be a dropdown with some that so the user can select instead off typing it.

![image](https://user-images.githubusercontent.com/16120290/169150123-8690672f-3dc0-4f13-a0ad-b33a2df24b01.png)

Thanks in advance

**Comment by macrozone** [comment, marked as answer]
hi @GuiPerim ,

in the schema (its a jsonschema), you can use enum to get a select field: https://github.com/react-page/react-page/blob/c77f12a1b4ba5294687ee9c4edddb11e72133f0b/examples/plugins/contactForm.tsx#L29

**Reply by GuiPerim** [reply]
Perfect, thats exactly what i want. Thank you!
